config/handle.yml

```diff
@@ -1,9 +1,11 @@
 development:
-  hostname: "localhost:3000"
-  email: "contact@example.com"
-  prefix: '1234.DEV'
-  admin: "1234.TEST/ADMIN"
-  index: 301
+  hostname: <%= ENV["HANDLE_TARGET_HOSTNAME"] || "localhost:3000" %>
+  email: <%= ENV["HANDLE_EMAIL"] || "contact@example.com" %>
+  prefix: <%= ENV["HANDLE_PREFIX"] || '1234.DEV' %>
+  admin: <%= ENV["HANDLE_ADMIN_PREFIX"] || "1234.TEST/ADMIN" %> # 0.NA/20.500.12101
+  index: <%= ENV["HANDLE_INDEX"] || 301 %> # 300
+  private_key: <%= ENV["HANDLE_PKEY_FILE"] || '' %>
+  pkey_passphrase: <%= ENV["HANDLE_PKEY_PASSPHRASE"] || '' %>
   secret_key: 'password'
```

Comment:
If we're going to support everything else with environments variables for security reasons, shouldn't `secret_key:` be dealt with in the same manner?

Reply:
Yes, except that we need the value to not be present at all to use `private_key`/`pkey_passphrase` authentication. I couldn't think of a way to do that without some amount of development that would seem wasted since we expect them to use key auth in production and `password` auth only exists to support testing.